### PR TITLE
remove ggplotify

### PR DESCRIPTION
**Pull request by mhallal1**
remove `ggplotify` from `Suggests` as it was added here: https://github.com/insightsengineering/teal.widgets/pull/119

### CODE REVIEW

**Review by arkadiuszbeer — APPROVED** [review]
LGTM

**Review by Polkas — APPROVED** [review]

**Review by Polkas — CHANGES_REQUESTED** [review]
please wait

File: R/plot_with_settings.R [hunk 1]
```diff
@@ -144,8 +144,9 @@ plot_with_settings_ui <- function(id) {
 #'       } else if (input$download_option == "trellis") {
 #'         lattice::densityplot(1)
 #'       } else if (input$download_option == "grob") {
-#'         tr_plot <- lattice::densityplot(1)
-#'         ggplotify::as.grob(tr_plot)
+#'         grid.newpage()
+#'         l <- linesGrob()
+#'         grid.draw(l)
```
Comment (Polkas) [inline R/plot_with_settings.R:149]:
```suggestion
#'         grid::grid.newpage()
#'         l <- grid::linesGrob()
#'         grid::grid.draw(l)
```

Grid is a base package but not the startup one.

**Review by Polkas — APPROVED** [review]
I had to add grid:: as it is not a startup package, but is a base one

**Pull request merged**